packages/neo-one-monitor/src/MonitorBase.js

```diff
@@ -721,6 +729,38 @@ export default class MonitorBase implements Span {
     }
   }
 
+  receive(report: Report): void {
```

Comment:
Maybe call this `report` since I suppose it could be used in either the browser or node for bulk metric addition.